### PR TITLE
#41: allow sourcemaps in development mode by default

### PR DESCRIPTION
**Pull request by l0rdn1kk0n**

### CODE REVIEW

File: library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java [hunk 1]
```diff
@@ -158,19 +165,25 @@ public WebjarsSettings cdnUrl(String cdnUrl) {
         return this;
     }
 
+    public WebjarsSettings sourcemaps(boolean active) {
```
Comment (martin-g) [inline library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java]: Why this is only in the impl ? 
I think it is fine to have it in the interface too.

**Comment by martin-g** [comment]
:+1:

File: library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java (hunk 1 above)
Comment (l0rdn1kk0n) [inline library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java]: the interface contains getters only at the moment.

Comment (martin-g) [inline library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java]: I see.

**Comment by martin-g** [comment]
Merged it to `wicket7` branch too!

**Pull request merged**